prospector.yml

```diff
@@ -1,9 +1,11 @@
-strictness: medium
+strictness: low
```

Comment:
I promised to return to this one: [The PR that I was working on](https://github.com/readthedocs/readthedocs.org/pull/10001) has an additional 20 issues to fix if we use the `medium` setting. They disappear when using `low`.

I think we should not upgrade this setting before we have a good strategy, otherwise it's gonna be really distracting.

```

Messages
========

readthedocs/builds/admin.py
  Line: 24
    pylint: missing-class-docstring / Missing class docstring
  Line: 74
    pylint: missing-class-docstring / Missing class docstring



Check Information
=================
         Started: 2023-03-16 17:24:15.313257
        Finished: 2023-03-16 17:24:22.591789
      Time Taken: 7.28 seconds
       Formatter: grouped
        Profiles: prospector.yml, full_pep8, doc_warnings, no_test_warnings, strictness_medium, strictness_high, strictness_veryhigh, no_member_warnings
      Strictness: from profile
  Libraries Used: django, celery
       Tools Run: dodgy, pep257, pep8, profile-validator, pyflakes, pylint
  Messages Found: 2

Messages
========

readthedocs/builds/models.py
  Line: 371
    pylint: signature-differs / Signature differs from overridden 'delete' method (col 4)
  Line: 409
    pylint: missing-function-docstring / Missing function or method docstring (col 4)
  Line: 508
    pylint: missing-function-docstring / Missing function or method docstring (col 4)
  Line: 608
    pylint: unused-argument / Unused argument 'args'
  Line: 791
    pylint: protected-access / Access to a protected member _config of a client class (col 16)
  Line: 825
    pylint: protected-access / Access to a protected member _config of a client class (col 30)
  Line: 963
    pylint: redefined-builtin / Redefining built-in 'type' (col 8)
  Line: 1167
    pylint: unused-argument / Unused argument 'kwargs'
  Line: 1187
    pylint: unused-argument / Unused argument 'version' (col 20)
    pylint: unused-argument / Unused argument 'match_arg' (col 29)
  Line: 1276
    pylint: signature-differs / Signature differs from overridden 'delete' method (col 4)
  Line: 1313
    pylint: missing-class-docstring / Missing class docstring
  Line: 1370
    pylint: missing-class-docstring / Missing class docstring

readthedocs/projects/models.py
  Line: 108
    pylint: signature-differs / Signature differs from overridden 'save' method (col 4)
  Line: 505
    pylint: signature-differs / Signature differs from overridden 'save' method (col 4)
  Line: 528
    pylint: signature-differs / Signature differs from overridden 'delete' method (col 4)
  Line: 1500
    pylint: missing-class-docstring / Missing class docstring
  Line: 1554
    pylint: missing-class-docstring / Missing class docstring
  Line: 1592
    pylint: too-many-locals / Too many local variables (19/15) (col 4)
  Line: 2118
    pylint: missing-class-docstring / Missing class docstring



Check Information
=================
         Started: 2023-03-16 17:24:15.303198
        Finished: 2023-03-16 17:24:30.662231
      Time Taken: 15.36 seconds
       Formatter: grouped
        Profiles: prospector.yml, full_pep8, doc_warnings, no_test_warnings, strictness_medium, strictness_high, strictness_veryhigh, no_member_warnings
      Strictness: from profile
  Libraries Used: celery, django
       Tools Run: dodgy, pep257, pep8, profile-validator, pyflakes, pylint
  Messages Found: 20

Check Information
=================
         Started: 2023-03-16 17:24:15.311702
        Finished: 2023-03-16 17:24:21.814440
      Time Taken: 6.50 seconds
       Formatter: grouped
        Profiles: prospector.yml, full_pep8, doc_warnings, no_test_warnings, strictness_medium, strictness_high, strictness_veryhigh, no_member_warnings
      Strictness: from profile
  Libraries Used: django, celery
       Tools Run: dodgy, pep257, pep8, profile-validator, pyflakes, pylint
  Messages Found: 0
```